lib/msal-common/src/logger/Logger.ts

```diff
@@ -62,7 +62,7 @@ export class Logger {
         const defaultLoggerCallback = () => {};
         this.localCallback = loggerOptions.loggerCallback || defaultLoggerCallback;
         this.piiLoggingEnabled = loggerOptions.piiLoggingEnabled || false;
-        this.level = loggerOptions.logLevel || LogLevel.Info;
+        this.level = this.validateLogLevel(loggerOptions);
```

Comment:
Would this work?
```suggestion
        this.level = typeof(loggerOptions.logLevel) === "number" ? loggerOptions.logLevel : LogLevel.Info;
```

Reply:
It works for non-numbers, but it fails to catch cases in which the logLevel number doesn't represent one of the established levels (i.e. anything under 0 and over 4 at the moment).  For numbers over 4, it would just default to `Trace` since it's the highest level.

Reply:
That's fine. We provide typings that codify the accepted values.